policyengine_us/parameters/gov/states/dc/tax/income/subtractions/disability_income_exclusion/cap.yaml

```diff
@@ -0,0 +1,18 @@
+description: DC caps the disability exclusion to this amount per disabled person.
+metadata:
+  unit: currency-USD
+  period: year
+  label: DC disability exclusion cap
+  reference:
+    # The legal code does not mention the specific exclusion amount
```

Comment:
Is this (a)(2)(M)? If so please cite it. Also look into the IRC history that subsection refers to. It may enable us to move these parameters to federal.